### PR TITLE
Update UI with doctors

### LINKED ISSUE #44: Edit the UI template to be able to show both Doctors and Patients
Follow the Figma mockup

### PR DESCRIPTION
**Pull request by Haozeee**
Added a doctors tab in the UI. For now, the two tabs can be switched by clicking it.

resolves #44

### CODE REVIEW

**Comment by Haozeee** [comment]
Screenshot 2021-10-05 at 20 17 25

**Comment by Haozeee** [comment]
The doctor's tab is empty for now 
Screenshot 2021-10-05 at 20 33 18

File: src/main/java/seedu/plannermd/ui/PersonTab.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        selectionModel = personTab.getSelectionModel();
+        initialiseTabPanels();
+    }
+
+    private void initialiseTabPanels() {
+        patientListPanel = new PatientListPanel(logic.getFilteredPatientList());
+        patientListPanelPlaceholder.getChildren().add(patientListPanel.getRoot());
+
+        // TODO pass in the filteredDoctorsList to the DoctorListPanel constructor
+        doctorListPanel = new DoctorListPanel(null);
```
Comment (Haozeee) [inline src/main/java/seedu/plannermd/ui/PersonTab.java:51]: The doctor list is null for now

File: src/main/java/seedu/plannermd/model/ModelManager.java [hunk 2]
```diff
@@ -23,6 +24,7 @@
     private final PlannerMd plannerMd;
     private final UserPrefs userPrefs;
     private final FilteredList<Patient> filteredPatients;
+    private PersonTabSwitcher personTabSwitcher;
```
Comment (Haozeee) [inline src/main/java/seedu/plannermd/model/ModelManager.java:27]: @hpkoh You can use this interface and its methods to switch between the Patient and Doctor tab.

**Review by hpkoh — APPROVED** [review]
LGTM 👍 
The current implementation as an API seems to work.
I was wondering if perhaps we can give `PersonTab` a copy of model instead and be sensitive to the state of model instead to separate the UI element more from the `ModelManager`

**Comment by Haozeee** [comment]
@hpkoh I think if you actually do this, it will make the UI stuff more interlinked with the ModelManager since now the UI tab has to keep track of the state of the model and respond to changes in Model.

**Review by jiayi1129 — APPROVED** [review]
LGTM! 👍 I like that you extended the `PatientCard` from `PersonCard`.

**Pull request merged**